### DISCUSSION TITLE
Why Yaml for config instead of TypeScript?

### DISCUSSION THREAD

**Discussion by nemanjam** [arthelokyo/astrowind#339] [General]
With TypeScript it would be easier to include environment variables into config and type and validate everything with Zod.

```bash
src/config.yaml
```

**Comment by prototypa** [comment]
Hi @nemanjam 

Can you give me more details or examples or show me a link to understand you better, please? I'm interested in what you're saying

I used yaml in AstroWind because it is easy to read, but I am not happy with the complexity that was generated around it.
